### DISCUSSION TITLE
Radio button not working with wire:model

### DISCUSSION THREAD

**Discussion by diegocaprioli** [livewire/livewire#8382] [1. Help] [answered]
Hello,

I'm having some issues with radio buttons. I made a very simple test code. In my blade template I have:

<input type="radio" wire:model="yesno" value="yes" /> YES
<input type="radio" wire:model="yesno" value="no" /> NO
<br/>
Value? = {{ $yesno }}

In my Livewire component I have:

...
public string $yesno = "no";
...

When I load the page in the browser, it properly shows "no" in "Value? = no". But as click and cycle from one to the other, it doesn't change the value.

Any ideas what I might be doing wrong? Thanks in advance!

**Comment by diegocaprioli** [comment]
Sorry, here is the proper formating for the code above:


I'm having some issues with radio buttons. I made a very simple test code. In my blade template I have:
```
<input type="radio" name="yesno" wire:model="yesno" value="yes" /> YES
<input type="radio" name="yesno" wire:model="yesno" value="no" /> NO
<br/>
Value? = {{ $yesno }}
```

In my Livewire component I have:
```
...
public string $yesno = "no";
...
```

When I load the page in the browser, it properly shows "no" in **"Value? = no**". But as click and cycle from one radio to the other, it doesn't change the value.

Any ideas what I might be doing wrong? Thanks in advance!

**Comment by danie-ramdhani** [comment, marked as answer]
You need to use `wire:model.live` if you want to see the live update

```blade
<input type="radio" name="yesno" wire:model.live="yesno" value="yes" /> YES
<input type="radio" name="yesno" wire:model.live="yesno" value="no" /> NO
<br/>
Value? = {{ $yesno }}
```
